site/reference/pkg/update/README.md

```diff
@@ -67,33 +43,47 @@ VERSION:
 
 ```
 --strategy:
-  Controls how changes to the local package are handled.  Defaults to fast-forward.
-
-    * resource-merge: perform a structural comparison of the original /
-      updated Resources, and merge the changes into the local package.
-    * fast-forward: fail without updating if the local package was modified
+  Defines which strategy should be used to update the package. This will change
```

Comment:
Does these descriptions match with flag descriptions in cobra code?

Reply:
No, but I don't think it is a goal that they are identical. This allows us to expand on the description compared to what we can do for the cobra flags, so they will almost certainly have details. I think the main question around this is how we want to expose information about flags in the cli. Do we use the description of flags from the reference docs or the cobra flags.

Reply:
@phanimarupaka I think we should sync up separately from this PR on how we should take the content from the site and make it available to as usage instructions in the CLI. If that sounds good to you, can you resolve the requested changes on this PR?

Reply:
Sounds good